neptune/new/handler.py

```diff
@@ -268,6 +269,12 @@ def log(
                 elif is_float_like(first_value):
                     attr = FloatSeries(self._container, parse_path(self._path))
                 elif is_string_like(first_value):
+                    simple_warning(
+                        "You're logging an object with an implicit cast to a string."
+                        " Use `.log(str(object))` instead."
+                        " We'll end support of this behavior in `neptune-client==1.0.0`.",
+                        stack_level=4,
```

Comment:
4? Czemu nie 2?

Reply:
It was based on my example (standard user-level logging with `.log`). I've added some +1 logic and will try to think it tomorrow